tests/solids/diamondC_2x1x1-Gaussian_pp/CMakeLists.txt

```diff
@@ -87,3 +123,15 @@ qmc_run_and_check(
   0
   DET_DIAMOND_SCALARS # VMC
 )
+
+qmc_run_and_check(
+  deterministic-diamondC_2x1x1_pp-vmcbatch_gaussian_sdj
+  "${qmcpack_SOURCE_DIR}/tests/solids/diamondC_2x1x1-Gaussian_pp"
```

Comment:
Can we also have vmcbatch_dmcbatch with 1 MPI 4 threads,  2 MPI 2 threads,  2 MPI 3 threads variants?